saleor/graphql/extensions/types.py

```diff
@@ -42,22 +43,26 @@ class Meta:
 
 
 class Plugin(CountableDjangoObjectType):
+    id = graphene.Field(type=graphene.ID, required=True)
     configuration = graphene.List(ConfigurationItem)
 
     class Meta:
         description = "Plugin."
         model = models.PluginConfiguration
         interfaces = [graphene.relay.Node]
-        only_fields = ["name", "description", "active", "configuration"]
+        only_fields = ["id", "name", "description", "active", "configuration"]
+
+    def resolve_id(self: models.PluginConfiguration, _info):
+        return self.name
```

Comment:
Not sure if this is in the scope of this PR, but don't we want to have slugs for this purpose?

Reply:
We should not depend on names being constant. I suggest introducing a Java-like reverse-domain identifier: `mirumee.avatax` or `saleor.tax.vatlayer`

Reply:
As agreed, we will return to this in another PR.